File: PROGRAMMERS/lv0/머쓱이보다 키 큰 사람.js
function solution(array, height) {
  return array.filter((i) => i > height).length;
}

let inp = [
  [[149, 180, 192, 170], 167],
  [[180, 120, 140], 190],
];

for (let i of inp) {
  console.log(solution(i[0], i[1]));
}

File: PROGRAMMERS/lv0/python/머쓱이보다 키 큰 사람.py
def solution(array, height):
    return len(list(filter(lambda x:x>height, array)))

inps = [
    [[149, 180, 192, 170], 167],
    [[180, 120, 140], 190]
]

for (array, height) in inps:
    print(solution(array, height))